src/StoryBoard.cpp

```diff
@@ -263,18 +289,26 @@ bool StoryBoard::readFile(const char *filename, ListMeta &LM, bool is_top) {
             // check action
             if ((j->action_id = getFunctID(j->action_name)) == 0) {
                 // warning message
-                std::cerr << "StoryBoard error: Action not defined: " << filename << " action " << j->action_name << " at line " << j->file_lineno
+                std::cerr << "StoryBoard error: Action " <<  j->action_name << " not defined: " << filename << " at line " << j->file_lineno
                           << " of " << i->file_name << std::endl;
             }
-#ifdef DGDEBUG
+#ifdef E2DEBUG
             std::cerr << thread_id << "Line " << j->file_lineno << " state is " << j->state << " actionid " << j->action_id
         << " listname " << j->list_name << std::endl;
 #endif
     }
 }
 // check for required functions
 
-return true;
+
+    for (std::vector<ListMeta::list_info>::iterator j = LMeta->list_vec.begin(); j != LMeta->list_vec.end(); j++) {
```

Comment:
Codacy found an issue: [Prefer prefix ++/-- operators for non-primitive types.](https://app.codacy.com/manual/numsys/e2guardian/pullRequest?prid=6238856)